.travis.yml

```diff
@@ -24,24 +24,24 @@ jobs:
      python: 3.4
      script: cd $(bash before_deploy.sh)
      deploy:
-     - provider: pypi
+     - provider: pypi  # dev
```

Comment:
I'm not sure why changes are showing up on this file...what does your git blame show?

Reply:
Those are Logan's changes. Not sure why git is showing these as diffs when master should already have them.

Reply:
rebased/merged